samples/createHttpTask.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2019, Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/**
+ * Create a task for a given queue with an arbitrary payload.
+ */
+async function createHttpTask(project, location, queue, url, options) {
+  // [START cloud_tasks_create_http_task]
+  // Imports the Google Cloud Tasks library.
+  const cloudTasks = require('@google-cloud/tasks');
+
+  // Instantiates a client.
+  const client = new cloudTasks.CloudTasksClient();
+
+  // TODO(developer): Uncomment these lines and replace with your values.
+  // const project = 'my-project-id';
+  // const queue = 'my-appengine-queue';
+  // const location = 'us-central1';
+  // const url = 'https://<project-id>.appspot.com/log_payload'
+  // const options = {payload: 'hello'};
+
+  // Construct the fully qualified queue name.
+  const parent = client.queuePath(project, location, queue);
+
+  const task = {
+    httpRequest: {
+      httpMethod: 'POST',
+      url: url, //The full url path that the request will be sent to.
+    },
+  };
+
+  if (options.payload !== undefined) {
+    task.httpRequest.body = Buffer.from(options.payload).toString(
+      'base64'
+    );
+  }
+
+  if (options.inSeconds !== undefined) {
+    task.scheduleTime = {
+      seconds: options.inSeconds + Date.now() / 1000,
+    };
+  }
+
+  const request = {
+    parent: parent,
+    task: task,
+  };
+
+  console.log('Sending task %j', task);
+  // Send create task request.
+  const [response] = await client.createTask(request);
+  const name = response.name;
+  console.log(`Created task ${name}`);
+
+  // [END cloud_tasks_create_http_task]
+}
+
+const cli = require(`yargs`)
```

Comment:
I believe we're dropping yargs for these in the rubric.  You can probably replace this all with:

```js
createHttpTask(...process.argv.slice(2)).catch(console.error);
```